packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -36,13 +36,14 @@ const modal_pages_indices = {
     restricted_country_signup_error: 9,
     invalid_input_error: 10,
 };
+let active_modal_index_no = 0;
```

Comment:
Tried moving to useState, however due to too many rerenders, react cancels the render and crashes the app to prevent
Needed to move the index out of the component to persist the state